src/api/baseapi.cpp

```diff
@@ -1015,6 +1015,7 @@ bool TessBaseAPI::ProcessPagesFileList(FILE *flist,
       return false;
     }
     tprintf("Page %d : %s\n", page, pagename);
+    fflush(stderr);
```

Comment:
I suggest to keep that code unchanged unless it's clear that the new code is really needed. As `stderr` is normally unbuffered, flushing should not be necessary.

Reply:
reverted change.